src/platform/static-data/error-messages.jsx

```diff
@@ -72,11 +73,11 @@ export const mhvAccessError = (
           </ol>
           <p>
             If none of these apply to you, please{' '}
-            <CallHelpDesk>
+            <SubmitSignInForm>
               call the {siteName}
               Help Desk at 1-855-574-7286, TTY: 1-800-877-8339. We're here
```

Comment:
Should all phone number be wrapped in anchor tags?

Reply:
Yes, but this one is going away soon (it's only visible on vets.gov environments, which are all deprecated).